crates/sui-core/src/consensus_adapter.rs

```diff
@@ -338,28 +347,30 @@ impl ConsensusAdapter {
             debug!("Submitting {transaction_key:?} to consensus");
             // We enter this branch when in select above await_submit completed and processed_waiter is pending
             // This means it is time for us to submit transaction to consensus
-            let _timer = self
-                .opt_metrics
-                .as_ref()
-                .map(|m| m.sequencing_acknowledge_latency.start_timer());
-            let mut retries = 0;
-            while let Err(e) = self
-                .consensus_client
-                .submit_to_consensus(&transaction, epoch_store)
-                .await
             {
-                // This can happen during Narwhal reconfig, so wait for a few retries.
-                if retries > 3 {
-                    error!(
-                        "Error submitting transaction to own narwhal worker: {:?}",
-                        e
-                    );
+                let _timer = self
+                    .opt_metrics
+                    .as_ref()
+                    .map(|m| m.sequencing_acknowledge_latency.start_timer());
```

Comment:
This will now measure the actual acknowledgment of the certificate from Narwhal. The overall sequencing latency will be measured separately by the `sequencing_certificate_latency` metric

Reply:
Can you move the comment on differences to metric description?

Reply:
Would it be possible to track the number of retries required as a label to be able to further breakdown the latency?

Reply:
Sure!